data/Remnant Cognizance.txt

```diff
@@ -0,0 +1,626 @@
+
+
+
+mission "Remnant: Cognizance 4"
+	name "Destroy Korath"
+	description "Prefect Chilia has asked you to eliminate the Korath fleet"
+	deadline 20
+	source "Caelian"
+	waypoint "Aescolanus"
+	to offer
+		has "Remnant: Cognizance 3: done"
+	on offer
+		conversation
+			`When you arrive in the spaceport you find Prefect Chilia surrounded by a group of Remnant. He is in the midst of a briefing, and waves you forward as soon as you approach. "Everyone, this is Captian <last>. They have located the Korath hunters twice now, so they will take the lead on tracking for this mission."`
+			`	He turns to you. "These are the captains who will be following your lead. You will have two Albatrosses, two Ibises with full complements, and 6 starlings. One of the Albatrosses is outfitted to be a shield to take the brunt of attacks." He pauses, and looks around the group. "This taskforce will be overkill for the intended target, but we are not taking chances: Make sure all the Korath die. Salvage is secondary, and there should be no escapees."`
+			`	He steps back and looks at the group. "May the Embers burn bright for you." With that, he gives the group a respectful nod and walks away. The captains all look around, make various signs you think are probably encouraging combat pilot slang, and variations on Prefect Chilia's parting words. As they disperse to their ships, one gestures to you. "Lead on, Captain."`
+				accept
+	npc kill
+		government "Korath"
+		personality staying
+		system "Nenia"
+		fleet
+			names "korath"
+			variant
+				"Korath World-Ship B (Ember)"
+					cargo "void sprite" 10 40
+				"Korath Raider (Ember)" 4
+					cargo "void sprite" 10 40
+		dialog "You have destroyed all the Korath that appeared to be hunting the Void Sprites."
+	npc
+		government "Remnant"
+		personality escort launching appeasing unconstrained disables plunders
+		fleet
+			names "remnant"
+			variant
+				"Albatross (Shield)"
+				"Albatross"
+				"Ibis" 2
+				"Starling (Heavy)" 6
+				"Petrel" 4
+				"Tern" 8
+	on enter "Aescolanus"
+		dialog `The Korath fleet is gone. Given the urgency Prefect Chilia put on this mission, the onus is on you to find it... Quickly.`
+	on complete
+		conversation
+			`The taskforce streams back into Caelian without fanfare. The ships settle onto their respective pads with a sense of satisfaction. Within seconds mechanics climbing the hull and engineers are dropping from the ceiling on tethers to inspect the engines. Batteries are plugged in, fuel lines hooked up, and carts with replacement weapnonry roll in. As always, no matter what they just did, the first responsibility of every captain is to ensure their ship is ready to head out. As the first footsteps echo on the hull of the <ship>, you switch the ship into mantenance mode before heading to the spaceport.`
```

Comment:
`The task force streams back into Caelian without fanfare, Its ships settle onto their respective pads with a sense of satisfaction. Within seconds mechanics climbing the hull and engineers are dropping from the ceiling on tethers to inspect the engines. Batteries are plugged in, fuel lines hooked up, and carts with replacement weapnonry roll in. As always, no matter what they did, the first responsibility of every captain is to ensure their ship is ready to head out. As the first footsteps echo on the hull of the <ship>, you switch the ship into mantenance mode before heading to the spaceport.`

Reply:
Thanks, fixed!